docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  postgres:
+    image: postgres:12-alpine
+    container_name: postgres
+    env_file:
+      - ./.env
+    volumes:
+      - data:/var/lib/postgresql/data

```

Comment:
I suppose `container_name` is not required if it is equal to the naming in 6th line.

Reply:
If container_name is not set, then on docker-compose up container will be named like <folder_name>_postgres_1